base/src/SimpleControlModule.cpp

```diff
@@ -0,0 +1,29 @@
+#include "SimpleControlModule.h"
+
+void SimpleControlModule::sendEOS()
+{
+	return Module::sendEOS();
+}
+
+void SimpleControlModule::sendEOS(frame_sp& frame)
+{
+	return Module::sendEOS(frame);
+}
+
+void SimpleControlModule::sendEOPFrame()
+{
+	return Module::sendEoPFrame();
+}
+
+// Right Now, Just Logging But Can be used to Do bunch of other things 
+void SimpleControlModule::handleError(const ErrorObject &error)
```

Comment:
i hope this method is virtual

Reply:
Yes it's a virtual function, defined in AbsControlModule